topics/chacha-cipher/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: chacha20, xchacha20, chacha12, xchacha12, chacha8, xchacha8, chacha20-poly1305, xchacha-poly1305
+display_name: ChaCha
+short_description: ChaCha is a family of stream ciphers developed by Daniel J. Bernstein. It's part of the popular authenicated encryption algorithm ChaCha20-Poly1305.
```

Comment:
```suggestion
short_description: ChaCha is a family of stream ciphers developed by Daniel J. Bernstein.
```
The test is failing because the short description is too long.

Reply:
Aha, MAX_SHORT_DESCRIPTION_LENGTH = 130.